node/test/tasklib.ts

```diff
@@ -19,47 +19,47 @@ var _testTemp = path.join(__dirname, '_temp');
 
 var plat = os.platform();
 
-var NullStream = function() {
+var NullStream = function () {
```

Comment:
all the formatting changes are OK but it makes focused changes really hard to review :)